software/known-issues.md

```diff
@@ -123,3 +131,107 @@ KNOWN_CURSORS=["HORIZON"]
 5. call "newdb" on the core instance to reset core's database
 6. start up stellar-core normally, wait for it to be in "Synced!" state
 7. start up Horizon, it should be able to see that core has data to ingest
+
+# Backfilling history
+
+You have a core instance that works properly (close ledgers), but you realize
+that for some reason its history is incomplete, in the sense that the oldest
```

Comment:
What does old enough mean? The genesis ledger? It feels like this should actually be defined.

Reply:
this is related to Horizon concepts (related to how ingestion works).
this document is probably not the right place to define those as they are generic